R/assertions.r

```diff
@@ -2,7 +2,10 @@
 NULL
 
 is.integerish <- function(x) {
-  is.integer(x) || (is.numeric(x) && all(x == as.integer(x)))
+  
+  # using trunc() to deal with very large numbers (including Inf)
+  res <- is.integer(x) || (is.numeric(x) && all(x == trunc(x)))
```

Comment:
Can you please eliminate the explicit return? See http://style.tidyverse.org/functions.html#return

Reply:
done